test/Steeltoe.Management.EndpointCore.Test/Security/AuthenticationTestMiddleware.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright 2017 the original author or authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Microsoft.AspNetCore.Http;
+using System;
+using System.Collections.Generic;
+using System.Security.Claims;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Steeltoe.Management.Endpoint.Security.Test
+{
+    internal class AuthenticationTestMiddleware
```

Comment:
Minor point, but it might be more illustrative to name this middleware something like `SetsUserInContextForTestsMiddleware` or just add a comment that briefly explains it so you don't have to read through the code and think about what it is for